docs/source/ns_blog.md

```diff
@@ -0,0 +1,53 @@
+# Neural Solution: Bring the Optimization Power of Intel® Neural Compressor as a Service
```

Comment:
The title is not catchy enough and needs to highlight the key points. It could be improved with a more attractive title